changelog.txt

```diff
@@ -11,6 +11,7 @@
 
 = 8.0.1 - 2025-02-13 =
 * Fix - Revert a change released in 7.2.0 which triggered the "woocommerce_cart_item_name" filter with the wrong number of parameters.
+* Fix - Removed unnecessary setting of renewal order paid date on status transition, relying on WooCommerce core behavior instead.
```

Comment:
This changelog needs to be moved up to the 8.1.0

Reply:
60a1d9d544ef9653656d044e68883162b1592a46 👍